### ISSUE TITLE
When a Zimit-based article is opened in a separate tab, external links and PDFs are blocked

### ISSUE THREAD

**Issue by Jaifroid** [kiwix/kiwix-js#1202]
See https://github.com/kiwix/kiwix-js-pwa/issues/540 for full explanation. Let's wait to see if zimit2 is affected, because this is an edge case in zimit classic caused by sandboxing of the replay_iframe.

**Comment by asmit27rai** [comment]
@Jaifroid I Want To Work On This Issue.

**Comment by Jaifroid** [comment]
@THEBOSS0369 Asked first on this. @THEBOSS0369 Are you still interested in this issue (to work on in January)?

**Comment by THEBOSS0369** [comment]
> [@THEBOSS0369](https://github.com/THEBOSS0369) Asked first on this. [@THEBOSS0369](https://github.com/THEBOSS0369) Are you still interested in this issue (to work on in January)?

@Jaifroid Yes sir i'm still very much interested in working on this issue.

**Comment by THEBOSS0369** [comment]
@Jaifroid Sir can you tell me whether this issue is still present or not i tried to recreate this issue but wasn't able to find it. Please see the screen recorder below
https://github.com/user-attachments/assets/46e079cb-9b15-499a-9968-8279845d3b99

If the issue is still present please let me know which zim i can use to recreate this issue.

**Comment by THEBOSS0369** [comment]
Hey @Jaifroid ! Sir let me know what to do further when you get free time. Thanks
